galasa-parent/dev.galasa/src/main/java/dev/galasa/api/run/Run.java

```diff
@@ -114,11 +112,11 @@ public String getObr() {
         return obr;
     }
 
-    public Boolean getLocal() {
+    public boolean getLocal() {
```

Comment:
isLocal()
isTraceEnabled() ?

Reply:
The trouble with this is we are changing the public API that other people may be using. We should deprecate the old method signature and add the newer one in instead... and test that they both do the same thing...

Reply:
Have left the old methods in referencing the new ones due to this issue, have left a comment in the old methods stating this.